clients/admin-ui/src/features/common/custom-fields/helpers.ts

```diff
@@ -6,3 +8,8 @@ export const getResourceType = (valueOrLabel: string) =>
   RESOURCE_TYPE_OPTIONS.find(
     (option) => option.value === valueOrLabel || option.label === valueOrLabel
   )?.value ?? ResourceTypes.SYSTEM;
+
```

Comment:
Oh, looks like `getResourceType` can be deleted now.

Reply:
Yes, this method can be deleted now given there are no references.